app/boot.coffee

```diff
@@ -8,7 +8,22 @@ global.cobudgetApp.run ($auth, CurrentUser, Dialog, LoadBar, $location, $q, Reco
 
   $rootScope.$on 'auth:validation-success', (ev, user) ->
     global.cobudgetApp.currentUserId = user.id
-    Records.memberships.fetchMyMemberships().then (data) ->
+    path_components = $location.path().split('/')
+    if path_components[1] == "groups"
+      groupId = path_components[2]
+    else if path_components[1] == "buckets"
+      bucketId = parseInt path_components[2]
+      console.log("bucketId")
+      console.log(bucketId)
+      Records.buckets.findOrFetchById(bucketId).then(bucket) ->
+        console.log("bucket")
+        console.log(bucket)
+        groupId = bucket.group().id
```

Comment:
this section starting on line 18 with `Records.buckets.findOrFetchById(bucketId).then(bucket)` isn't getting executed, but i am not sure why. perhaps you can't have a promise inside another promise? @jlopker i am guessing you might know what is happening here better as i am just poking around in the dark and making shit up basically :).